torch/_torch_docs.py

```diff
@@ -2951,6 +2951,51 @@ def merge_dicts(*dicts):
 
 """.format(**common_args))
 
+add_docstr(torch.logaddexp,
+           r"""
+logaddexp(input, other, out=None) -> Tensor
+
+Logarithm of the sum of exponentiations of the inputs.
+
+Calculates pointwise :math:`\log\left(e^x + e^y\right)`. This function is useful
+in statistics where the calculated probabilities of events may be so small as to
+exceed the range of normal floating point numbers. In such cases the logarithm
+of the calculated probability is stored. This function allows adding
+probabilities stored in such a fashion.
+
+This op should be disambiguated with :func:`torch.logsumexp` which do a
+reduction with a tensor over a specific axis.
+
+Args:
+    {input}
+    other (Tensor): the tensor to compute logaddexp with
+    {out}
+
+Example::
+
+    >>> torch.logaddexp(torch.tensor([-1.0]), torch.tensor([-1.0, -2, -3]))
```

Comment:
These examples are helpful but I think you only need a single example showing the addition of multiple small probabilities, like the NumPy documentation, since that's the scenario that logaddexp is designed for.

Reply:
it is intended, just to show the numerical stability